### ISSUE TITLE
proposals quickly goes to zero

### ISSUE THREAD

**Issue by zmonoid** [marvis/pytorch-yolo2#105]
Any suggestions?

**Comment by beckhamchen** [comment]
It doesn't matter. It will increase after training one epoch

**Comment by sheepdnb** [comment]
Check your learning rate, batch size etc. It'll go back up with after an epoch or two if your hyperparameters are set correctly. I'm using batch size = 16, learning rate = 0.0001 and not having any issues with proposals.
